src/StoreKeeper/WooCommerce/B2C/Tools/OrderHandler.php

```diff
@@ -92,4 +103,23 @@ public function addShopProductIdsToMetaData($order, $data)
 
         $order->add_meta_data(self::SHOP_PRODUCT_ID_MAP, $id_map);
     }
+
+    protected function isSyncAllowed(int $orderId): bool
+    {
+        if (EventsHandler::isEventsDisabled('orders')) {
+            return false;
+        }
+
+        if (0 !== $orderId) {
```

Comment:
what are the cases when `$orderId === 0` ?

Reply:
Removed checking of order ID and changed logic to positive checking.